docs/guides/dev_guide.md

```diff
@@ -83,103 +87,109 @@ e94346bce41402155ef120e2525fad2d0bf30b10a89e4b93fd8471df1e6a0981
 
-A `message` can be up to 32 kb in size and it can hold variable sets of information called `payloads`. The number of payloads a single message can encapsulate is not given. Even a message without a `payload` is completely valid and can be broadcast. The `Message` itself does not include any timestamp; a message timestamp is derived from an acceptance of the given message by the Tangle network.
+A `message` can be up to 32 kb in size and it can hold variable sets of information called `payloads`. The number of payloads a single message can encapsulate is not given. Even a message without a `payload` is completely valid and can be broadcast. The `message` itself does not include any timestamp; a message timestamp is derived from an acceptance of the given message by the Tangle network.
 
-More information: [Protocol-rfc#0017 - Message](https://github.com/GalRogozinski/protocol-rfcs/blob/message/text/0017-message/0017-message.md) 
+For further refefence, please see our [Protocol-rfc#0017 - Message](https://github.com/GalRogozinski/protocol-rfcs/blob/message/text/0017-message/0017-message.md) 
 
 A`payload` represents a layer of concern. Some payloads may change the state of the ledger (ex. `SignedTransactions`) and some may provide extra features to some specific applications and business use cases (ex. `IndexationPayload`).
 
 There are already implemented core payloads, such as `SignedTransaction`, `MilestonePayload`, and `IndexationPayload` but the message and payload definition is generic enough to incorporate any future payload(s) the community agrees upon.
 
-More information: [Protocol-rfc#0018 - Transaction Payload](https://github.com/luca-moser/protocol-rfcs/blob/signed-tx-payload/text/0000-transaction-payload/0000-transaction-payload.md)
+For further refefence, please see our: [Protocol-rfc#0018 - Transaction Payload](https://github.com/luca-moser/protocol-rfcs/blob/signed-tx-payload/text/0000-transaction-payload/0000-transaction-payload.md)
 
 Needless to say, the IOTA network ensures the outer structure of the message itself is valid and strictly complies with network consensus protocol. However, the inner structure is very flexible, future-proof, and offers an unmatched network extensibility.
 
 ![messages_in_tangle](assets/messages_in_tangle.svg)
 
-The current IOTA 1.5 network incorporates the following core payloads:
-* `SignedTransaction`: a payload that describes `UTXO` transactions that are the cornerstones of value-based transfers in the IOTA network. Via this payload, a `message` can be also cryptographically signed
-* `MilestonePayload`: a payload that is emitted by the Coordinator
+The current Chrysalis network incorporates the following core payloads:
+* `SignedTransaction`: a payload that describes `UTXO` transactions that are the cornerstones of value-based transfers in the IOTA network. Via this payload, a `message` can be also cryptographically signed.
```

Comment:
* `SignedTransaction`: a payload that describes `UTXO` transactions, which are the cornerstones of value-based transfers in the IOTA network. Via this payload, a `message` can be also cryptographically signed.